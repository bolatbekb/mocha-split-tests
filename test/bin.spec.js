"use strict";

const util = require("util");
const exec = util.promisify(require("child_process").exec);

describe("bin/mocha-split-tests script", function() {
  it("can be executed", async function() {
    const { stdout, stderr } = await exec(
      `./bin/mocha-split-tests -t 3 -g 2 -r test/data/fakeSpecs/runtime.log -f 'test/data/fakeSpecs/**/*.spec' -s ';'`
    );
    expect(stdout).toMatchSnapshot();
  });
});
