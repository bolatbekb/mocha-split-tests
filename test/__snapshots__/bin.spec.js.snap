// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bin/mocha-split-tests script can be executed 1`] = `
"test/data/fakeSpecs/2.spec;test/data/fakeSpecs/specs/1.spec
"
`;
